ring-core/src/ring/middleware/file.clj

```diff
@@ -40,14 +40,20 @@
   Accepts the following options:
 
   :index-files?    - look for index.* files in directories, defaults to true
-  :allow-symlinks? - serve files through symbolic links, defaults to false"
+  :allow-symlinks? - serve files through symbolic links, defaults to false
+  :handler-first? - let handler process the files first. if handler successfully processes the file, do not serve static content"
```

Comment:
I think `prefer-handler?` might be a better name for the option. Please also ensure that the docstring wraps at 80 characters and has correct grammar.

Something more like:

    :prefer-handler?  - prioritize handler response over files, defaults to false